financial-connections/src/main/java/com/stripe/android/financialconnections/di/FinancialConnectionsSheetNativeModule.kt

```diff
@@ -53,11 +54,20 @@ internal interface FinancialConnectionsSheetNativeModule {
 
     @Singleton
     @Binds
-    fun providesNavigationManager(
+    fun bindsNavigationManager(
         impl: NavigationManagerImpl
     ): NavigationManager
 
     companion object {
+
+        @Provides
+        @Singleton
+        fun providesCustomTabsManager(
+            logger: Logger,
+        ): CustomTabsManager = CustomTabsManager(
```

Comment:
Hmm is this a common pattern used across FC? it's a bit confusing to me at first sight before looking into the `invoke`, could we directly call the `CustomTabsManagerImpl` directly?

Reply:
The idea is to keep `CustomTabsManagerImpl` internal in `stripe-core` to prevent exposing the implementation. Happy to expose it if that does not sound good.

Reply:
I don't see any issues making the `CustomTabsManagerImpl` being exposed with `RestrictTo`, just looking at the dagger code here it looks antipattern, feels like `CustomTabsManager` is a concrete class which could just declare `@Inject` in its constructor

Reply:
https://github.com/stripe/stripe-android/pull/7081/commits/1f7b1cbfbc45922ce8478b420df2d51429dd284e